lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -98,9 +128,29 @@ local function onDrop( ply, wep )
     setSpeedFromWeight( ply, totalWeight )
 end
 
+local function onPlayerSay( ply, text )
+    if not IsValid( ply ) then return end
+    if not ply:Alive() then return end
+    
+    if isDropCommand[text] then 
+        dropPlyWeapon( ply )
+        return ""
```

Comment:
Is this how we keep it from going to chat?

Reply:
Yes, http://wiki.garrysmod.com/page/GM/PlayerSay it says so in the return section.

Reply:
Neat!